xml/OrganisationType.xml

```diff
@@ -32,34 +34,39 @@
             <name>
                 <narrative>Other Public Sector</narrative>
                 <narrative xml:lang="fr">Autre secteur public</narrative>
+                <narrative xml:lang="es">Otras entidades del sector público</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>21</code>
             <name>
                 <narrative>International NGO</narrative>
                 <narrative xml:lang="fr">ONG internationale</narrative>
+                <narrative xml:lang="es">ONG internacional</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>22</code>
             <name>
                 <narrative>National NGO</narrative>
                 <narrative xml:lang="fr">ONG nationale</narrative>
+                <narrative xml:lang="es">ONG nacional</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>23</code>
             <name>
                 <narrative>Regional NGO</narrative>
                 <narrative xml:lang="fr">ONG regionale</narrative>
+                <narrative xml:lang="es">ONG regional</narrative>
             </name>
         </codelist-item>
         <codelist-item>
             <code>24</code>
             <name>
                 <narrative>Partner Country based NGO</narrative>
                 <narrative xml:lang="fr">ONG dans le pays bénéficiaire</narrative>
+                <narrative xml:lang="es">ONG con sede en el país asociado</narrative>
```

Comment:
This should possibly be:
> ONG con sede en el país beneficiario de la ayuda

Not sure the language "partner country" is used in this way in Spanish